readme.md

```diff
@@ -213,6 +213,19 @@ Default: `process.cwd()`
 
 Current working directory of the project to retrieve the diagnostics for.
 
+##### typingsFile
+
+Type: `string`<br>
+Default: `''`
+
+Path to the type definitions of the project.
+
+##### testFiles
+
+type: `string[]`<br>
+default: `['']`
+
+An array of test files with their path
```

Comment:
Same as above (needs to answer what its purpose is) and you're missing a dot at the end.